cmd/auth.go

```diff
@@ -31,7 +32,8 @@ func newLoginCommand(astroClient astro.Client, out io.Writer) *cobra.Command {
 			return login(cmd, args, astroClient, out)
 		},
 	}
-	cmd.Flags().BoolVarP(&loginLink, "login-link", "l", false, "Get login link to login on a separate device for browserless cloud CLI login")
+	cmd.Flags().BoolVarP(&loginLink, "login-link", "l", false, "Get login link to login on a separate device for cloud CLI login")
+	cmd.Flags().BoolVarP(&tokenLogin, "token-login", "t", false, "Login with a token for browserless cloud CLI login")
```

Comment:
same comment as before, I think it'd be clearer to rename tokenLogin to shouldLoginWithToken